src/test/java/com/olgaboiar/mint/MockRouteMap.java

```diff
@@ -0,0 +1,37 @@
+package com.olgaboiar.mint;
+
+import com.olgaboiar.mint.handlers.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MockRouteMap implements IRouteMap {
+    private Map<String, Map<String, IHandler>> routes = new HashMap<>();
+    MockRoutesConfiguration serverTestRoutes;
+
+    public MockRouteMap(MockRoutesConfiguration serverTestRoutes) {
+        this.serverTestRoutes = serverTestRoutes;
+        registerTestRoutes();
+    }
+
+    @Override
+    public Map<String, Map<String, IHandler>> getMap() {
+        return routes;
+    }
+
+    @Override
+    public void registerTestRoutes() {
+        ArrayList<Route> allRoutes = serverTestRoutes.getAllRoutes();
+        for (Route r : allRoutes) routes.put(r.getPath(), r.getAllowedMethods());
```

Comment:
I prefer always spell out the variable.

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/551796ba1cf74690588733bdd9a165f44c6a9bb1